.yamato/urp_template-win-dx12.yml

```diff
@@ -12,8 +12,8 @@ URP_Template_Win_DX12_playmode_mono_Linear_trunk:
     agent:
         type: Unity::VM::GPU
         image: automation-tooling/win-10:v0.0.8-1149381
-        flavor: b1.xlarge
-        model: rtx2080
+        flavor: b1.large
+        model: gtx1080
```

Comment:
Is changing models related to xlarge to large change?

Reply:
Answered in DM :)